noxfile.py

```diff
@@ -23,7 +23,7 @@ def tests(session):
     """
     Run the unit and regular tests.
     """
-    session.install("-r", "requirements-test.txt", "./awkward-cpp", "./awkward")
+    session.install("-r", "requirements-test.txt", "./awkward-cpp", ".")
```

Comment:
This needs to be in two lines, I think.

```suggestion
    session.install("-r", "requirements-test.txt", "./awkward-cpp")
    session.install(".")
```